rules/REQUEST-901-INITIALIZATION.conf

```diff
@@ -98,23 +107,23 @@ SecRule &TX:early_blocking "@eq 0" \
     ver:'OWASP_CRS/3.4.0-dev',\
     setvar:'tx.early_blocking=0'"
 
-# Default Paranoia Level (rule 900000 in crs-setup.conf)
-SecRule &TX:paranoia_level "@eq 0" \
+# Default Blocking Paranoia Level (rule 900000 in crs-setup.conf)
+SecRule &TX:BLOCKING_PARANOIA_LEVEL "@eq 0" \
```

Comment:
Should the variable name be lowercase here?

Reply:
I guess converting to lower case would keep the naming more consistent with the rest of the file. On the other hand, converting all variable names in this file to upper case would be more consistent wit the rest of CRS.

@fzipi & @dune73 What do you think?

Reply:
Hmmm.. this is our paragraph for this about naming conventions in the https://github.com/coreruleset/coreruleset/blob/v3.4/dev/CONTRIBUTING.md#variable-naming-conventions

Reply:
I changed it to lower case.